toolchain/sem_ir/ids.h

```diff
@@ -141,6 +141,12 @@ struct ConstantId : public IdBase, public Printable<ConstantId> {
     return InstId(Abs(index) - IndexOffset);
   }
 
+  // Returns the instruction that describes this constant value, or
+  // InstId::Invalid if this is invalid or a runtime value.
+  constexpr auto inst_id_if_valid() const -> InstId {
```

Comment:
Feels kind of odd to be adding this for one use, although presumably you'll add more...

I should ask in particular though, why `constexpr`?

Reply:
I just made this `constexpr` because `inst_id` is, and it seemed reasonable for it to be consistent.

Reply:
FWIW, I'd thought we were only doing constexpr when it was used that way, rather than simple consistency.